keyboards/anavi/macropad8/readme.md

```diff
@@ -0,0 +1,16 @@
+# nano
+
+![nano](https://1.bp.blogspot.com/-goa_eXx1McA/WEhvoSdfDSI/AAAAAAAB_bM/XdstcXe_eKkQxuYcNI5cTySTVo3xZxaeQCEw/s640/IMG_20161207_101051.jpg)
```

Comment:
The nano actaully already exists in the repo (under `/keyboards/40percentclub/nano`). 

If this is just a modification of that, then this would be better as a keymap under that, rather than as a separate board.